Resources/views/show_recommendations.html.twig

```diff
@@ -15,6 +15,7 @@
                     outputTypeId: '{{ outputTypeId }}',
                     contextItems: '{{ contentId }}',
                     categoryPath: '{{ categoryPath }}',
+                    filters: '{{ filters is defined ? filters: '' }}',
```

Comment:
Small nitpick: `{{ filters is defined ? filters : '' }}` (add space between `filters` and a semicolon)